assembly/assembly-main/src/assembly/bin/che.sh

```diff
@@ -739,4 +743,4 @@ if [ "${USE_HELP}" == "false" ] && [ "${JUMP_TO_END}" == "false" ]; then
       launch_che_server
     fi
   fi
-fi
+fi
```

Comment:
Please do not remove trailing new line in files. It will cause changes in diffs on each commit